gabimoreno/src/main/res/values/strings.xml

```diff
@@ -40,4 +40,7 @@
     <string name="notifications_rationale">La app requiere notificaciones para avisarte cuando aparezca un nuevo audio</string>
     <string name="course_thumbnail">Carátula del audiocurso</string>
     <string name="course_active">Audiocurso activo</string>
+    <string name="course_listened">Audiocurso escuchado</string>
+    <string name="course_link">Obten el audiocurso completo</string>
```

Comment:
We are not using the billing library yet. This would break the Google Play policies.

Reply:
I would put this, @kmorfo:
<string name="course_link">Accede al audiocurso completo</string>

If we put only "Accede" it is not clear enough.